python.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external python 2.7.6
+### RPM external python 2.7.10
```

Comment:
With the bootstrap bundle already built for OSX10.10 maybe we can leave this unmerged.